### ISSUE TITLE
Font Display problem

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#1551]
Sample 1551 sent by email
From forum https://forum.sumatrapdfreader.org/t/some-problem-to-display-a-pdf/2841/5
Included Monospaced fonts appear distorted

Fonts: ArialMT (Type1)
AvantGardeITCbyBT-Book (Type1)
Calibri (TrueType)
Calibri-Bold (TrueType)
Monospace821BT-Bold (Type1)
Monospace821BT-BoldItalic (Type1)
Monospace821BT-Italic (Type1)
Monospace821BT-Roman (Type1)

Problem with Monospace
![image](https://user-images.githubusercontent.com/12754913/79130778-7128e900-7d9f-11ea-82ef-d57749a1d963.png)

**If I substitute Courier for Monospaced and _oblique for italic_ etc. the problems are reduced**

![image](https://user-images.githubusercontent.com/12754913/79130606-23ac7c00-7d9f-11ea-8b95-0ed5e553dc66.png)

Fonts: ArialMT (Type1)
AvantGardeITCbyBT-Book (Type1)
Calibri (TrueType)
Calibri-Bold (TrueType)
Courier (Type1)
Courier-Bold (Type1)
Courier-BoldOblique (Type1)
Courier-Oblique (Type1)

**Comment by kjk** [comment]
```
loading non-embedded font 'Calibri' from 'C:\Windows\Fonts\calibri.ttf'
loading non-embedded font 'Calibri-Bold' from 'C:\Windows\Fonts\calibrib.ttf'
couldn't find system font 'AvantGardeITCbyBT-Book'
couldn't find system font 'Monospace821BT-Italic'
couldn't find system font 'Monospace821BT-BoldItalic'
couldn't find system font 'Monospace821BT-Roman'
couldn't find system font 'Monospace821BT-Bold'
loading non-embedded font 'ArialMT' from 'C:\Windows\Fonts\arial.ttf
```

This is a bad PDF - it refers to a non-standard font and doesn't include it. It seems we pick up `ArialMT` a substitute font and it has different metrics.

mupdf-gl.exe has the same problem.

**Comment by Yackor** [comment]
Hello, 
this pdf is displayed correctly with Adobe and Firefox reader ...

**Comment by GitHubRulesOK** [comment]
@Yackor 
As described the problem is that the document was written on a system that had a set of fonts that don't normally exist on Windows machines (they may be found on some machines owned by graphics artists or fontographers) but not on Joe Publics PC

It does not help that when the fonts are not included in a PDF then it is up to each user's installation to make its own best educated guesswhat each caracter looks like. Even downloading the named font is not good enough.

So in Firefox it loads a seriffed MonoSpace font **which looks close enough**. **in my Acrobat it loads a Sans Serif MM which is nothing like the original** and in SumatraPDF it sees ArialMT as the nearest request which produces poor results

In short we have an english saying RIRO (rubbish in rubbish out.)

**Comment by GitHubRulesOK** [comment]
@Yackor 
Just to confirm that if embedded per the PDF standards then SumatraPDF will show fonts "alien" to windows This was my test sheet to show why Lucida Console is the best match for a non embedded Windows replacement to Monospace821
[Document Mono-OK.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/4486396/Document.Mono-OK.pdf)

**Comment by GitHubRulesOK** [comment]
Closing as duplicate of several other cases such as Reader is unable to show cyrillic text in pdf file #378 or `SumatraPDF showing garbled texts in this file` #1585 
Where files may only be viewed easily within a locality or machine that has a specific font available However the PDF is not defined using globally available fonts or the font glyphs have not been named/embedded in a rational way, thus breaking the "Portable"DF standard requirements.

Note most of these examples can be easily fixed in the editor to be made acceptable to all viewers
